linux/xdg/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    java
+    kotlin("jvm")
+    kotlin("kapt")
+}
+
+dependencies {
+    implementation(projects.autoDarkModeBase)
+    implementation(libs.darklaf.nativeUtils)
+
+    implementation("com.github.hypfvieh:dbus-java:3.3.1")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.6.1")
+
```

Comment:
Please declare those in https://github.com/weisJ/auto-dark-mode/blob/master/gradle/libs.versions.toml and use the type safe accessors.